sandbox.html

```diff
@@ -15,13 +15,32 @@ <h1 v-if="message.length % 2 == 0">{{ message }}</h1>
         <button @click="list.push(list.length + 1)">Push Number</button>
         <button @click="list.pop()">Pop Number</button>
         <button @click="list.reverse()">Reverse List</button>
+        <button @click="show1 = !show1">Character List</button>
+        <button @click="show2 = !show2">Favourite list</button>
+        <button @click="favlist.push(prompt('Enter favourite character:'))">Add Favourite Character</button>
 
         <ul v-if="show && list.length">
             <li v-for="item of list">{{ item }}</li>
         </ul>
 
-  <p v-else-if="list.length">List is not empty, but hidden.</p>
-  <p v-else>List is empty.</p>
+    <p v-else-if="list.length">List is not empty, but hidden.</p>
+    <p v-else>List is empty.</p>
+      
+
+        <ul v-if="show1 && charalist.length">
+            <li v-for="item of charalist">{{ item }}</li>
+        </ul>
+
+    <p v-else-if="charalist.length">Character List is not empty, but hidden.</p>
+    <p v-else>Character List is empty.</p>
+
```

Comment:
Somewhere in between - it's missing to add characters in the general list - it's direct fav list - well the concept is clear